lib/transproc/array/combine.rb

```diff
@@ -5,8 +5,9 @@ class Combine
 
       class << self
         def combine(array, mappings)
+          return EMPTY_ARRAY if array.empty?
           root, nodes = array
-          return root if nodes.nil?
+          return root if nodes.nil? || nodes.empty?
```

Comment:
unfortunately we need to use `nodes.count.zero?` here, because there's no `Relation::Loaded#empty?` in rom (I actually added it to master but it's too early to start relying on it here, we can clean it up later)